src/tests/integration/test_foundry_prove.py

```diff
@@ -757,6 +757,7 @@ def test_foundry_xml_report(
         tests=[
             ('AssertTest.test_assert_true()', None),
             ('AssertTest.test_assert_false()', None),
+            ('AssertNestedTest.test_assert_true_nested()', None),
         ],
         prove_options=ProveOptions(
```

Comment:
@palinatolmach I think we can remove the `reinit` flag now. We can do that in this PR.

Reply:
Sure, I agree! The line to be removed is:
https://github.com/runtimeverification/kontrol/blob/134b0a407385b769727bc55f291c3164782ee119/src/tests/integration/test_foundry_prove.py#L763

Reply:
Removed! :slightly_smiling_face: